src/community.md

```diff
@@ -8,36 +8,33 @@ eleventyNavigation:
 
 Welcome to the Kuadrant community, where we're working to bring together Gateway API and Open Cluster Management to help you manage, scale, load-balance and secure your Ingress Gateways as a key part of your application connectivity, in a single cluster or the multi-cluster Kubernetes environments.
 
-### Events and Meetings Calendar
+<!--- Need to add html for headings as markdown does not have native support for creating a heading with an anchor--->
```

Comment:
Hmm...that's interesting. The only difference I can see is that this example is being served via docs.kuadrant.io and pulled in externally from the docs repository whereas the contributors guide is being served directly from the website i.e. this repository. @jasonmadigan any thoughts?

Reply:
kuadrant.io uses eleventy, docs.kuadrant.io uses mkdocs (which does render MD with anchors for headings).

Eleventy here is using `markdownIt` - it may be possible that it can generate GH style markdown with anchors too. Will take a quick look.

Reply:
We can probably use https://github.com/valeriangalliat/markdown-it-anchor

Reply:
@pmccarthy added here: https://github.com/pmccarthy/kuadrant.github.io/pull/1

Reply:
Great, thanks Jason!

Reply:
Branch updated now based on Jason's changes